interface/main/messages/messages.php

```diff
@@ -878,6 +880,35 @@ function deselectRow(row) {
                 ,minDate : 0 //only future
             })
 
+            <?php if ($GLOBALS['phimail_enable']) : ?>
+            $('.trusted-messages-force-check').click(function() {
+                window.top.restoreSession();
+                request = new FormData;
+                request.append("skip_timeout_reset", "1");
```

Comment:
if this function is being run manually, then can remove above line. If this function is done automatically on a timer, then would leave in.
(this basically bypasses the timeout mechanism (logout user that is not active for a certain amount of time), so that automated scripts do not impact the timeout (where the timeout would never happen))

Reply:
ok, no timeout, I just grabbed the sample from what we were doing in the main window frame for running the background service.

Reply:
Fixed.